### DISCUSSION TITLE
How do I fall back to the default configuration from `nvim-lspconfig`?

### DISCUSSION THREAD

**Discussion by nalzok** [williamboman/nvim-lsp-installer#553] [Q&A] [answered]
So I have installed the `texlab` language server with `nvim-lsp-installer`, and the process is very smooth! However, `texlab` requires quite a bit of configuration, and [the config file](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/server_configurations/texlab.lua) from `nvim-lspconfig` seems like a sensible default to me. I could just copy & paste their config to my `init.lua`, but that doesn't seem like a proper approach. Can I fall back to the default config provided by `nvim-lspconfig`?

As a bonus question, how do I inspect the setting for my current language server?

**Comment by williamboman** [comment, marked as answer]
Hello! The default config provided by nvim-lspconfig will actually be applied by default. Are you not experiencing this?

**Reply by nalzok** [reply]
Thanks, but I cannot confirm that because I don't know any method to view the setting for the current language server. Any thoughts?

**Reply by williamboman** [reply]
You can access the applied settings of a server like so. The index (`[1]`) you want to access will probably differ, depending on how many servers you're running.

```
:lua =vim.lsp.get_active_clients()[1].config.settings
```

<sup>btw `:lua =<lua_expr>` is shorthand for `:lua print(vim.inspect(<lua_expr>))`</sup>
